lambda/main.tf

```diff
@@ -37,6 +37,7 @@ resource "aws_lambda_function" "lambda_function" {
 resource "aws_cloudwatch_log_group" "lambda_log_group" {
   name              = "/aws/lambda/${aws_lambda_function.lambda_function.function_name}"
   retention_in_days = var.log_retention
+  kms_key_id        = var.log_group_kms_key
```

Comment:
Standardise naming convention for this field? `log_group_kms_key` vs `environment_variables_key_arn`

Reply:
I've gone for `environment_variables_kms_key_arn` and `log_group_kms_key_arn`